src/parser/hn_parser.py

```
@@ -3,9 +3,7 @@
 """
 HNレコードパーサー: １８．繁殖馬マスタ
 
-このファイルは自動生成されました。
-Source: 公式JV-Data仕様書 Ver.4.9.0.1
-Generated by: scripts/generate_all_parsers.py
+Source: 公式JV-Data仕様書 Ver.4.9.0.1 ＋ 2026-08-14 の BLDN 実測
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `＋` (FULLWIDTH PLUS SIGN). Did you mean `+` (PLUS SIGN)?

(RUF002)